.travis.yml

```diff
@@ -39,14 +29,22 @@ env:
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*2of2"
     - SBT_CMD="mavenResolverPluginTest:scripted dependency-management/*1of2 project/transitive-plugins"
 
-notifications:
-  email:
-    - eugene.yokota@typesafe.com
-    - joshua.suereth@typesafe.com
+matrix:
+  fast_finish: true
+  include:
+    - env: SBT_CMD="scalafmtDiffTest"
```

Comment:
how about `git diff --exit-code` ;), quick, simple, efficient. After running a normal format that is.